docs/Help.md

```diff
@@ -68,18 +68,28 @@ If you're getting the `Login expired for user <user>! Try adding your account ag
 ### How do I add Chatterino as an OBS dock?
 This is not natively supported, but there is a work around:
 
-1. Add a dock into OBS for standard Twitch chat
-2. Turn on `Always on top` in Chatterino
-3. Click the 3 dots up the top of a channel split and click `Popup`
-4. Move and resize the chat to overlay on top of the standard Twitch chat
+1. Add a dock into OBS for standard Twitch chat.
+2. Turn on `Always on top` in Chatterino.
+3. Click the 3 dots up the top of a channel split and click `Popup`.
+4. Move and resize the chat to overlay on top of the standard Twitch chat.
 
-### How do I delete the Chatterino cache?
+### Where is Chatterino folder located?
+On **Windows**:
+`%APPDATA%/Chatterino2`
 
-For **Windows**:
-Delete the `%APPDATA%/Chatterino2/Cache` folder
+On **Linux**:
+`$HOME/.local/share/chatterino`
 
-For **Linux**:
-Delete the `$HOME/.local/share/chatterino/Cache` folder: `rm -rf "$HOME/.local/share/chatterino/Cache"`
+On **Mac**:
+`$HOME/Library/Application Support/chatterino`
 
-For **Mac**:
-Delete the `$HOME/Library/Application Support/chatterino/Cache` folder: `rm -rf "$HOME/Library/Application Support/chatterino/Cache"`
+### How do I delete the Chatterino cache / settings?
+Navigate to [Chatterino folder](#where-is-chatterino-folder-located) and remove corresponding folders.
```

Comment:
I'm not sure about this, it's making a simple step more complicated for no reason. Where a simple troubleshooting step such as "Delete your Chatterino cache" can be used and easily found, they now have to go to two different locations to find out what they have to delete.